src/transformers/modeling_dpr.py

```diff
@@ -0,0 +1,551 @@
+# coding=utf-8
+# Copyright 2018 DPR Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch DPR model for Open Domain Question Answering."""
+
+
+import collections
+import logging
+from typing import List, Optional, Tuple
+
+import torch
+from torch import Tensor as T
+from torch import nn
+from torch.serialization import default_restore_location
+
+from .configuration_dpr import DprConfig
+from .file_utils import add_start_docstrings
+from .modeling_bert import BertConfig, BertModel
+from .modeling_utils import PreTrainedModel
+
+
+logger = logging.getLogger(__name__)
+
+DPR_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "dpr-base-uncased",
```

Comment:
same here `facebook/dpr-base-uncased`

Reply:
should I put all three of them ?

Reply:
all three models? I would put every model that can be download from the model hub here